user-manuals/README.md

```diff
@@ -90,7 +90,8 @@ Video: Application Instance
 
 **Related flows:**
 
-1. [User management](https://app.gitbook.com/o/l53nGvPQLhOHrKCP9HTG/s/vsbL9DiW8kFNTItD6870/)
-2. [Configurations](https://app.gitbook.com/o/l53nGvPQLhOHrKCP9HTG/s/PoTFyo0cPPGdyZu8kxeb/)
+1. [Launchpad](https://launchpad.hotwax.io/home/)
+2. [User management](https://app.gitbook.com/o/l53nGvPQLhOHrKCP9HTG/s/vsbL9DiW8kFNTItD6870/)
```

Comment:
These are internal links to gitbook backend. need to use links for front end. Please fix these as well @sanjanashrivastava